src/components/CardItem/CardItem.tsx

```diff
@@ -0,0 +1,106 @@
+'use client';
+
+import { Typography, Box } from '@mui/material';
+import Image from 'next/image';
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+
+interface CardItemInterface {
+  item: {
+    id: string;
+    imageSrc: string;
+    title: string;
+    text: string;
+  };
+  type: 'WhyUs' | 'Possibilities';
```

Comment:
change to other names, cardItem must include information and logic that refers to cardItem, 'WhyUs' and 'Possibilities' is not a part of cardItem, its a part of website, for example you can change possible types to 'small', 'large' and etc.